apis/config/v1beta1/configuration_types.go

```diff
@@ -307,6 +307,9 @@ type Integrations struct {
 	//  - "kubeflow.org/xgboostjob"
 	//  - "pod"
 	Frameworks []string `json:"frameworks,omitempty"`
+	// List of GroupVersionKinds that are managed for Kueue by external controllers
+	// the expected format of the string is "$GROUP/$VERSION, Kind=$KIND"
```

Comment:
Can we use the format that this supports instead? https://github.com/kubernetes/apimachinery/blob/37988e577e160bf7749437141fee1d6b8ca15dd6/pkg/runtime/schema/group_version.go#L44